change: user account and access page (#2003)
plentymarkets/plenty-manual-docs: docs/de-de/modules/business-entscheidungen/pages/benutzerkonten-zugaenge.adoc

In der Benutzer:innen-Verwaltung erstellst du neue Benutzer:innen-Konten und bearbeitest bestehende Benutzer:innen-Konten. Darüber hinaus verknüpfst du Benutzer:innen-Konten mit Rollen und Zugängen.

> * Mit Rollen gruppierst du Benutzer:innen-Konten und ordnest dieselben Berechtigungen zu mehreren Benutzer:innen-Konten.

Reviewer: ```suggestion
* Mit Rollen gruppierst du Benutzer:innen-Konten und ordnest mehreren Konten dieselben Berechtigungen zu.
```

[#35]
== Sicherheitseinstellungen

Im Menü *Einrichtung » Kontverwaltung » Sicherheit* stellst du die Anforderungen ein, die Benutzer:innen erfüllen müssen, wenn sie ein neues Passwort vergeben.

[.instruction]
Sicherheitseinstellungen vornehmen:

. Öffne das Menü *Einrichtung » Kontoverwaltung » Sicherheit*.

> . Nimm die Einstellungen vor. Beachte dazu die Erläuterungen in <<tabelle-sicherheit-einstellen>>.

Reviewer 2: ```suggestion
. Nimm die Einstellungen vor. Beachte dazu die Erläuterungen in <<#tabelle-sicherheit-einstellen>>.
```

|Anzahl der zuletzt benutzten Passwörter einstellen, die nicht erneut verwendet werden dürfen. Die Nutzer:in wird darauf hingewiesen, wenn sie versucht, eines der zuletzt benutzten Passwörter erneut einzugeben. Wenn diese Funktion nicht gewünscht ist, 0 eingeben oder leer lassen. +
*_Hinweis:_* Die Anzahl der zu speichernden Passwörter muss 20 oder weniger betragen.

| *Maximaler Zeitraum der Passwortgültigkeit (Tage)*
|Nach Ablauf des Gültigkeitszeitraums in Tagen wird das Benutzer:innen-Konto gesperrt. Wenn diese Funktion nicht gewünscht ist, lass den Regler bei Null stehen.

| *Passwort-Ablaufbenachrichtigung (Tage)*
|Gib mit dem Regler die Anzahl an Tagen an.
So viele Tage vor Ablauf des Passworts erhält der:die Benutzer:in eine tägliche Erinnerung zur Änderung des Passworts.

> Wenn diese Funktion nicht gewünscht ist, lass den Regler bei Null stehen.

Reviewer 2: ```suggestion
Wenn diese Funktion nicht gewünscht ist, lasse den Regler bei 0 stehen.
```

[.instruction]
Benutzer:innen-Konto bearbeiten:

. Öffne das Menü *Einrichtung » Kontoverwaltung » Konten*.
. Klicke auf *Suchen* (material:search[]). +
→ Alle Benutzer:innen-Konten werden angezeigt.

> . Nimm Änderungen und Einstellungen gemäß <<tabelle-benutzer:innen-daten>> vor.

Reviewer 2: ```suggestion
. Nimm Änderungen und Einstellungen gemäß <<#tabelle-benutzer:innen-daten>> vor.
```

[[tabelle-benutzer:innen-daten]]
.Daten eines Benutzer:innen-Kontos
[cols="1,3"]
|====
|Einstellung |Erläuterung

| *Benutzer*
|Name ohne Leerzeichen eingeben. Der Name ist eine Pflichtangabe, die für den Login benötigt wird. Der Benutzername darf kein Leerzeichen enthalten und muss mindestens 5 Zeichen lang sein.

| *Name*
|Realer Name der Benutzer:in, die in der Kopfzeile des plentymarkets Administrationsbereichs angezeigt wird.

| *E-Mail*
|E-Mail-Adresse des Benutzer:innen-Kontos eingeben. Diese ist u.a. für Benachrichtigungen des Systems, Benachrichtigungen aus Ereignisaktionen und auch für die Gruppenkommunikation innerhalb des Systems wichtig.

| *Zeitzone*
|Zeitzone des Benutzer:innen-Kontos wählen.

| *Sprache*
|Sprache für den Admin-Bereich. Diese Sprache wird als Standard beim Login verwendet.

| *Passwort*
|Neues Passwort eingeben, um dem Benutzer:innen-Konto ein neues Passwort zu erteilen.

| *Passwort wiederholen*
|Passwort noch einmal eingeben.

| *App-Pin*
|include::app:partial$app-pin.adoc[]

| *App-Pin wiederholen*
|PIN noch einmal eingeben.

| *IP beschränken*

> |IP-Adresse des Rechners der Person des Benutzer:innen-Kontos eingeben, damit sich die Person mit diesem Benutzer:innen-Konto nur über diese IP-Adresse in das plentymarkets System einloggt.

Reviewer: ```suggestion
|IP-Adresse des Rechners der Person des Benutzer:innen-Kontos eingeben, damit sich die Person mit diesem Benutzer:innen-Konto nur über diese IP-Adresse in das plentymarkets System einloggen kann.
```

|*Aktiviere Forum-Konto*
|Klicken, wenn für das Benutzer:innen-Konto ein Forum-Konto erstellt werden soll. Dafür muss eine E-Mail-Adresse hinterlegt sein.

| *Benutzer deaktiviert*
|Aktivieren, um das Benutzer:innen-Konto zu deaktvieren. Deaktivierte Personen können sich nicht mehr in das System einloggen.

| *Benutzerzugang*

> |Benutzer:innen-Zugang des Benutzer:innen-Kontos wählen. Es stehen die Zugänge *Administrator*, und *Nur API-Benutzer* zur Verfügung.

Reviewer: Hier würde ich noch mal wiederholen, was die Zugänge für Auswirkungen haben. Vor allem Administrator sollte ja nicht leichtfertig vergeben werden.

| *Zugewiesene Rollen*
|Rollen des Benutzer:innen-Kontos wählen. Damit Rollen zur Auswahl stehen, müssen zuvor xref:business-entscheidungen:benutzerkonten-zugaenge.adoc#70[Rollen] erstellt werden.

| *Signatur*

> | Text, der deinen Nachrichten und Kommentaren im Ticketsystem automatisch als Signatur angefügt wird.

Reviewer: ```suggestion
| Text, der Nachrichten und Kommentaren dieser Benutzer:in im Ticketsystem automatisch als Signatur angefügt wird.
```

|====

[#40]
=== Eigner aktivieren

> Ob und in welchen Bereichen das Benutzer:innen-Konto als Eigner verfügbar sein soll, wird über bestimmte Berechtigungen im erweiterten Modus der Rollen bestimmt. Wenn du zum Beispiel eine Rolle erstellst, die Eigner-Rechte für den Wareneingang verfügt musst du im erweiterten Modus *Warenbestände* oder gezielter *Warenbestände » Wareneingänge* als Berechtigung freigeben. Welche Berechtigungen welche Eignerrechte aktiviert findest du unter <<tabelle-eignerrechte>>. Wenn das Benutzer:innenkonto über Adminrechte verfügt, verfügt es auch über alle Eignerrechte.

Reviewer 2: ```suggestion
Ob und in welchen Bereichen das Benutzer:innen-Konto als Eigner verfügbar sein soll, wird über bestimmte Berechtigungen im erweiterten Modus der Rollen bestimmt. Wenn du zum Beispiel eine Rolle erstellst, die Eignerrechte für den Wareneingang haben soll, musst du im erweiterten Modus *Warenbestände* oder gezielter *Warenbestände » Wareneingänge* als Berechtigung freigeben. Welche Berechtigungen welche Eignerrechte aktiviert, findest du unter <<#tabelle-eignerrechte>>. Wenn das Benutzer:innenkonto über Adminrechte verfügt, verfügt es auch über alle Eignerrechte.
```

> Ob und in welchen Bereichen das Benutzer:innen-Konto als Eigner verfügbar sein soll, wird über bestimmte Berechtigungen im erweiterten Modus der Rollen bestimmt. Wenn du zum Beispiel eine Rolle erstellst, die Eigner-Rechte für den Wareneingang verfügt musst du im erweiterten Modus *Warenbestände* oder gezielter *Warenbestände » Wareneingänge* als Berechtigung freigeben. Welche Berechtigungen welche Eignerrechte aktiviert findest du unter <<tabelle-eignerrechte>>. Wenn das Benutzer:innenkonto über Adminrechte verfügt, verfügt es auch über alle Eignerrechte.

Reviewer: ```suggestion
Ob und in welchen Bereichen das Benutzer:innen-Konto als Eigner verfügbar sein soll, wird über bestimmte Berechtigungen im erweiterten Modus der Rollen bestimmt. Wenn du zum Beispiel eine Rolle erstellst, die über Eigner-Rechte für den Wareneingang verfügen soll, musst du im erweiterten Modus *Warenbestände* oder gezielter *Warenbestände » Wareneingänge* als Berechtigung freigeben. Welche Berechtigungen welche Eignerrechte aktivieren findest du in <<tabelle-eignerrechte>>. Wenn das Benutzer:innenkonto über Adminrechte verfügt, verfügt es auch über alle Eignerrechte.
```

[.instruction]
Benutzer:innen als Eigner aktivieren:

. Öffne das Menü *Einrichtung » Kontoverwaltung » Rollen*.
. Öffne eine Rolle.

> . Wechsel auf den erweiterten Modus (material:toggle_on[]).

Reviewer: ```suggestion
. Wechsel in den erweiterten Modus (material:toggle_on[]).
```

[#85]
== Mehrere Benutzer:innen-Konten bearbeiten

Mehrere Benutzer:innen-Konten bearbeitest du über *Mehr* (material:more_vert[]) im Menü *Konten*. Hierbei werden zuvor gewählte Benutzer:innen-Konten bearbeitet.

[.instruction]
Gruppenfunktion ausführen:

. Öffne das Menü *Einrichtung » Kontoverwaltung » Konten*.
. Wähle die zu bearbeitenden Benutzer:innen.
. Klicke auf material:more_vert[] *Mehr*.
. Nimm die Einstellungen vor. Beachte die Erläuterungen in <<#tabelle-mehr>>.
. Klicke auf *Anwenden*. +
→ Die Benutzer:innen-Konten werden geändert.

[[tabelle-mehr]]
.Benutzer:innen-Konten per *Mehr* bearbeiten
[cols="1,3"]
|====
|Einstellung |Erläuterung

| *E-Mail senden*
|Aktivieren, um eine E-Mail an die gewählten Benutzer:innen-Konten zu senden. Das E-Mail-Programm wird geöffnet. Die Benutzer:innen-Konten sind als Empfänger:in eingetragen.

| *Angefangene Box-Kommissionierungen zurücksetzen*
| Aktivieren, um die angefangene Box-Kommissionierung der ausgewählten Benutzer:innen-Kontos zurückzusetzen.

| *Ausloggen*
|Aktivieren, um ausgewählte Benutzer:innen aus dem System auszuloggen.
|====

[#105]
== Rechte verwalten

> In plentymarkets werden Rechte über Rollen an Benutzer:innen vergeben. Dies beinhaltet Menü- und Plugin-Sichtbarkeiten sowie REST-API-Berechtigungen. Ein:e Benutzer:in hat also immer die Rechte, die von der Rolle zugewiesen werden. Die einzige Ausnahme dabei bilden Administrators, die unabhängig von ihrer Rolle alle Rechte im System besitzen.

Reviewer 2: ```suggestion
In plentymarkets werden Rechte über Rollen an Benutzer:innen vergeben. Dies beinhaltet Menü- und Plugin-Sichtbarkeiten sowie REST-API-Berechtigungen. Ein:e Benutzer:in hat also immer die Rechte, die von der Rolle zugewiesen werden. Die einzige Ausnahme dabei bilden Administratoren, die unabhängig von ihrer Rolle alle Rechte im System besitzen.
```

Im Menü *Einrichtung » Kontoverwaltung » Rollen* kannst du neue Rollen erstellen oder existierende Rollen angepassen. Beim Erstellen oder Bearbeiten einer Rolle kannst du im Menüpunkt *Zugriffsrechte* die Auftragsherkünfte freigeben, die für die Benutzer:innen sichtbar sein sollen. Rollen können von Benutzer:innen des Zugangs *Administrator* erstellt werden oder von Nutzer:innen, die eine Rolle mit den notwendigen Rechten haben. +
Ordne einem Benutzer:innen-Konto nur die Rolle zu, die die Person für ihren Aufgabenbereich benötigt. Diese Einschränkung erhöht die Sicherheit und beugt Fehlern in deinem System vor.

> Die Tabelle der Benutzerrechte gliedert sich in vier Spalten. In diesen Spalten kannst du Rechte vergeben, indem du die Optionen *Betrachten*, *Bearbeiten*, *Erstellen* oder *Löschen* aktivierst. Zwei dieser vier Optionen ändern im erweiterten Modus ihre Namen: Aus *Betrachten* wird *Lesen* und aus *Bearbeiten* wird *Aktualisieren*.

Reviewer 2: ```suggestion
Die Tabelle der Benutzer:innenrechte gliedert sich in vier Spalten. In diesen Spalten kannst du Rechte vergeben, indem du die Optionen *Betrachten*, *Bearbeiten*, *Erstellen* oder *Löschen* aktivierst. Zwei dieser vier Optionen ändern im erweiterten Modus ihre Namen: Aus *Betrachten* wird *Lesen* und aus *Bearbeiten* wird *Aktualisieren*.
```

→ Dein plentymarkets Vertrag wird angepasst.

[#50]
=== Persönliche Einstellungen

> Unter dem plenty p und deinem Profilbild findest du die *persönlichen Einstellungen*. Hier nimmst du Einstellungen vor, die nur für das im System angemeldete Benutzer:innen-Konto gelten. Deshalb ist das Tab für dich auch nur in deinem eigenen Benutzer:innen-Konto sichtbar. Die persönlichen Einstellungen unterteilen sich in die Tabs xref:willkommen:system-personalisieren.adoc#70[Allgemein], und xref:willkommen:system-personalisieren.adoc#80[Shortcuts] die nur dich persönlich betreffen.

Reviewer 2: ```suggestion
Unter dem plentymarkets Logo oben links und deinem Profilbild oben rechts findest du die *persönlichen Einstellungen*. Hier nimmst du Einstellungen vor, die nur für das im System angemeldete Benutzer:innen-Konto gelten. Deshalb ist das Tab für dich auch nur in deinem eigenen Benutzer:innen-Konto sichtbar. Die persönlichen Einstellungen unterteilen sich in die Tabs xref:willkommen:system-personalisieren.adoc#70[Allgemein], und xref:willkommen:system-personalisieren.adoc#80[Shortcuts] die nur dich persönlich betreffen.
```

> Unter dem plenty p und deinem Profilbild findest du die *persönlichen Einstellungen*. Hier nimmst du Einstellungen vor, die nur für das im System angemeldete Benutzer:innen-Konto gelten. Deshalb ist das Tab für dich auch nur in deinem eigenen Benutzer:innen-Konto sichtbar. Die persönlichen Einstellungen unterteilen sich in die Tabs xref:willkommen:system-personalisieren.adoc#70[Allgemein], und xref:willkommen:system-personalisieren.adoc#80[Shortcuts] die nur dich persönlich betreffen.

Reviewer: ```suggestion
Unter dem plentymarkets p und deinem Profilbild findest du die *persönlichen Einstellungen*. Hier nimmst du Einstellungen vor, die nur für das im System angemeldete Benutzer:innen-Konto gelten. Deshalb ist das Tab für dich auch nur in deinem eigenen Benutzer:innen-Konto sichtbar. Die persönlichen Einstellungen unterteilen sich in die Tabs xref:willkommen:system-personalisieren.adoc#70[Allgemein], und xref:willkommen:system-personalisieren.adoc#80[Shortcuts] die nur dich persönlich betreffen.
```

[.instruction]
Gruppenfunktion ausführen:

. Öffne das Menü *Einrichtung » Kontoverwaltung » Konten*.

> . Wähle die zu bearbeitenden Benutzer:innen.

Reviewer 2: ```suggestion
. Wähle (material:check_box[role=skyBlue]) die zu bearbeitenden Benutzer:innen.
```

> . Klicke auf *Gruppenfunktion*. +

Reviewer 2: Das passende Icon ist dann material:more_vert[].

|====
|Einstellung |Erläuterung

> | *Mail senden*

Reviewer 2: Kann das bitte in der UI umbenannt werden in "E-Mail senden"?

Author: Nehme das morgen mit ins Daily, denke das sollte kein Problem sein

|Aktivieren, um E-Mail an die gewählten Benutzer:innen-Konten zu senden. Das E-Mail-Programm wird geöffnet. Die Benutzer:innen-Konten sind als Empfänger:in eingetragen.

| *Angefangene Box-Kommisionierungen zurücksetzen*
| Aktivieren, um die angefangene Box-Kommissionierung der ausgewählten Benutzer:innen-Kontos zurückzusetzen.

| *Ausloggen*
|Aktivieren, um ausgewählte Benutzer:innen aus dem System auszuloggen.
|====

[#105]
== Rechte verwalten

> In plentymarkets werden Rechte über Rollen an Benutzer:innen vergeben. Dies beinhaltet Menü- und Plugin-Sichtbarkeiten sowie REST-API-Berechtigungen. Ein:e Benutzer:in hat also immer die Rechte, die von der Rolle zugewiesen werden. Die einzige Ausnahme dabei bilden Admins, die unabhängig von ihrer Rolle, alle Rechte im System besitzen.

Author: Letzteres!

> Im Menü *Einrichtung » Kontoverwaltung » Rollen* können neue Rollen erstellt werden oder existierende Rollen angepasst werden. Beim Erstellen oder Bearbeiten einer Rolle kann im Menüpunkt *Zugriffsbeschränkung* die Auftragsherkünfte freigegeben werden, die für die Benutzer:innen sichtbar sein sollen. +

Reviewer 2: Das soll dann "Zugriffsrechte" heißen, oder?

Author: Ja genau! Heißt es auch schon, hab das hier wohl überlesen

Ordne einem Benutzer:innen-Konto nur die Rolle zu, die die Person für ihren Aufgabenbereich benötigt. Diese Einschränkung erhöht die Sicherheit und beugt Fehlern in deinem System vor.

> Die Tabelle der Benutzerrechte gliedert sich in vier Spalten in denen Rechte vergeben werden können durch das Anhaken einer Checkbox: *Betrachten*, *Bearbeiten*, *Erstellen*, *Löschen*. Zwei dieser vier Arten ändern im erweiterten Modus ihre Namen, aus *Betrachten* wird *Lesen* und aus *Bearbeiten* wird *Aktualisieren*.

Reviewer: ```suggestion
Die Tabelle der Benutzerrechte gliedert sich in vier Spalten. In diesen Spalten kannst du Rechte vergeben, indem du die Optionen *Betrachten*, *Bearbeiten*, *Erstellen* oder *Löschen* aktivierst. Zwei dieser vier Optionen ändern im erweiterten Modus ihre Namen: Aus *Betrachten* wird *Lesen* und aus *Bearbeiten* wird *Aktualisieren*.
```

[#106]
=== Einfacher und erweiterteter Modus

> Beim Erstellen und Bearbeiten von Rollen gibt es zwei Modi, den einfachen und den erweiterten. Welcher Modus für welche Rolle relevant ist, kommt auf die Rechte an, die der Rolle zugewiesen werden sollen. Der einfache Modus und erweiterter Modus sind verknüpft. Wenn du also Menü-Sichtbarkeiten im einfachen Modus auswählst werden automatisch Berechtigungen im erweiterten Modus automatisch aktiv sein. Diese Verknüpfung existier allerdings nur diese Richtung, wenn du also im erweiterten Modus Berechtigungen vergibst werden keine Menü- oer Plug-In-Sichtbarkeiten automatisch aktiv sein.

Reviewer 2: ```suggestion
Beim Erstellen und Bearbeiten von Rollen gibt es zwei Modi: den einfachen Modus und den erweiterten Modus. Welcher Modus für welche Rolle relevant ist, kommt auf die Rechte an, die der Rolle zugewiesen werden sollen. Der einfache Modus und der erweiterte Modus sind verknüpft. Wenn du also Menü-Sichtbarkeiten im einfachen Modus auswählst, werden automatisch Berechtigungen im erweiterten Modus automatisch aktiv sein. Diese Verknüpfung existiert allerdings nur in diese Richtung: Wenn du also im erweiterten Modus Berechtigungen vergibst, werden keine Menü- oder Plugin-Sichtbarkeiten automatisch aktiv sein.
```

> Beim Erstellen und Bearbeiten von Rollen gibt es zwei Modi, den einfachen und den erweiterten. Welcher Modus für welche Rolle relevant ist, kommt auf die Rechte an, die der Rolle zugewiesen werden sollen. Der einfache Modus und erweiterter Modus sind verknüpft. Wenn du also Menü-Sichtbarkeiten im einfachen Modus auswählst werden automatisch Berechtigungen im erweiterten Modus automatisch aktiv sein. Diese Verknüpfung existier allerdings nur diese Richtung, wenn du also im erweiterten Modus Berechtigungen vergibst werden keine Menü- oer Plug-In-Sichtbarkeiten automatisch aktiv sein.

Reviewer: ```suggestion
Beim Erstellen und Bearbeiten von Rollen gibt es zwei Modi: Den einfachen und den erweiterten Modus. Welcher Modus für welche Rolle relevant ist, kommt auf die Rechte an, die der Rolle zugewiesen werden sollen. Der einfache Modus und der erweiterte Modus sind miteinander verknüpft. Wenn du also Menü-Sichtbarkeiten im einfachen Modus aktivierst, sind diese Berechtigungen auch im erweiterten Modus automatisch aktiv. Diese Verknüpfung existiert allerdings nur in diese Richtung. Wenn du also im erweiterten Modus Berechtigungen vergibst, werden keine Menü- oder Plug-In-Sichtbarkeiten automatisch aktiviert.
```

> **Einfacher Modus** = Im einfachen Modus kannst du die Sichtbarkeiten für Menüs und Plug-Ins vergeben. Die Tabelle des einfachen Modus ist genauso aufgebaut wie die Navigation und bildet zudem individuelle Plug-Ins ab.

Reviewer 2: ```suggestion
**Einfacher Modus** = Im einfachen Modus kannst du die Sichtbarkeiten für Menüs und Plugins vergeben. Die Tabelle des einfachen Modus ist genauso aufgebaut wie die Navigation und bildet zudem individuelle Plugins ab.
```

> **Einfacher Modus** = Im einfachen Modus kannst du die Sichtbarkeiten für Menüs und Plug-Ins vergeben. Die Tabelle des einfachen Modus ist genauso aufgebaut wie die Navigation und bildet zudem individuelle Plug-Ins ab.

Reviewer: ```suggestion
**Einfacher Modus** = Im einfachen Modus kannst du die Sichtbarkeiten für Menüs und Plugins vergeben. Die Tabelle des einfachen Modus ist genauso aufgebaut wie die Navigation und bildet zudem individuelle Plugins ab.
```

> **Erweiterter Modus** = Im erweiterteten Modus vergibst du ausschließlich Berechtigungen.

Reviewer: Was ist der Unterschied zwischen Sichtbarkeiten und Berechtigungen?

[#110]
=== Rollen vergeben

Eine Rolle vergibst du an ein Konto wie folgt:

. Öffne das Menü *Einrichtung » Kontoverwaltung » Konten*.
→ Die Übersicht wird geöffnet.
. Wähle ein Konto.
→ Die Konto-Detailansicht wird geöffnet.

> . Wähle im Reiter *Rechte* das Dropdown-Menü *Zugewiesene Rollen*

Reviewer 2: ```suggestion
. Klicke im Bereich *Rechte* auf die Dropdown-Liste *Zugewiesene Rollen*.
```

> . Wähle im Reiter *Rechte* das Dropdown-Menü *Zugewiesene Rollen*

Reviewer: ```suggestion
. Wähle im Tab *Rechte* das Dropdown-Menü *Zugewiesene Rollen*.
```

> → Das Dropdown-Menü klappt aus.

Reviewer 2: ```suggestion
→ Die Dropdown-Liste klappt aus.
```

> → Das Dropdown-Menü klappt aus.

Reviewer: ```suggestion
→ Das Dropdown-Menü wird ausgeklappt.
```

> . Wähle eine vorhandene Rolle oder erstelle eine neue auf *Neue Rolle erstellen ...* (material:add[]).

Reviewer: ```suggestion
. Wähle eine vorhandene Rolle. +
*_Oder:_* Erstelle eine neue Rolle. Klicke dazu auf *Neue Rolle erstellen ...* (material:add[]).
```

> Sobald du die Rolle an das Konto vergeben hast, erhält das Konto die dazugehörigen Rechte.

Reviewer: ```suggestion
→ Das Konto erhält die Rechte der Rolle.
```

[#115]
=== Rechte für Rollen vergeben

[.instruction]
Rechte für Rollen vergeben:

. Öffne das Menü *Einrichtung » Kontoverwaltung » Rollen*. +
→ Die Übersicht wird geöffnet.

> . Klicke auf material:search[], um alle Rollen zu sehen.

Reviewer 3: fehlt hier noch das keyword "Suchen"? In der nächsten Config weiter unten benutzt du das. Consistency.

. Wähle eine Rolle. +

> → Der Einfache Modus mit den Rechten wird geöffnet.

Reviewer 3: Hier stolper ich ein bisschen drüber; I may be pedantic though.
Ist der einfach Modus dasjenige, was geöffnet wird? Oder wird nicht eher etwas anderes (Übersicht der Rechte/Rollen?) *im* einfachen Modus geöffnet?

Author: Du hast schon Recht, ist die elegantere Lösung!

. Vergib die Rechte für Rollen in dem jeweiligen Bereich durch Anklicken der Checkboxen. Aktiviere komplette Bereiche oder klappe Bereiche auf, um einzelne Berechtigungen zu aktivieren, die dem Benutzer:innen-Konto zur Verfügung stehen sollen.
. *Speichere* (icon:save[]) die Einstellungen.

[#116]
=== Rechte für rollendefinierte Ansichten vergeben

> In der Rollen-Detailansicht können Admins im Tab *Rollendefinerte Ansichten* bestimmen, welche Ansichten Benutzer:innen einer Rolle sehen und verwenden dürfen. Am Anfang ist immer die Standardansicht vorausgewählt. Existiert eine weitere Ansicht, kann die Standardansicht für eine Rolle deaktiviert und Rechte für eine andere Ansicht können zugewiesen werden. Beachte, dass immer mindestens eine Ansicht ausgewählt sein muss.

Reviewer 2: ```suggestion
In der Rollen-Detailansicht können Admins im Tab *Rollendefinierte Ansichten* bestimmen, welche Ansichten Benutzer:innen einer Rolle sehen und verwenden dürfen. Am Anfang ist immer die Standardansicht vorausgewählt. Existiert eine weitere Ansicht, kann die Standardansicht für eine Rolle deaktiviert und Rechte für eine andere Ansicht können zugewiesen werden. Beachte, dass immer mindestens eine Ansicht ausgewählt sein muss.
```

[.instruction]
Rechte für rollendefinierte Ansichten vergeben:

. Öffne das Menü *Einrichtung » Kontoverwaltung » Rollen*. +
→ Die Übersicht wird angezeigt.
. Klicke auf *Suchen* (icon:search[]), um nach Rollen zu suchen. +
→ Alle vorhandenen Rollen werden angezeigt.
. Wähle eine Rolle. +
→ Die Übersicht wird geöffnet.

> . Wähle das Tab *Rollendefinierte Ansichten*. +

Reviewer 2: ```suggestion
. Klicke auf den Bereich *Rollendefinierte Ansichten*. +
```

→ Alle rollendefinierten Ansichten werden angezeigt.
. Vergib die Rechte für Ansichten durch Anklicken der Checkboxen.
. *Speichere* (icon:save[]) die Einstellungen. +
→ Die Rechte für die gewählten Ansichten werden der Rolle zugewiesen.

[#100]
=== Zugriffsrechte festlegen

> Unter *Einrichtung » Kontoverwaltung » Rollen » [Rolle auswählen]* legst du fest, welche Bereiche und Einstellungen die Person mit der jeweiligen Rolle sehen kann, wenn die Person ein Menü öffnet. Bei fehlender Berechtigung ist zum Beispiel ein Lager oder sind Aufträge in einem Status für diese Person nicht sichtbar.

Reviewer 2: ```suggestion
Unter *Einrichtung » Kontoverwaltung » Rollen » [Rolle wählen]* legst du fest, welche Bereiche und Einstellungen die Person mit der jeweiligen Rolle sehen kann, wenn die Person ein Menü öffnet. Bei fehlender Berechtigung ist zum Beispiel ein Lager oder sind Aufträge in einem Status für diese Person nicht sichtbar.
```

> Unter *Einrichtung » Kontoverwaltung » Rollen » [Rolle auswählen]* legst du fest, welche Bereiche und Einstellungen die Person mit der jeweiligen Rolle sehen kann, wenn die Person ein Menü öffnet. Bei fehlender Berechtigung ist zum Beispiel ein Lager oder sind Aufträge in einem Status für diese Person nicht sichtbar.

Reviewer: ```suggestion
Unter *Einrichtung » Kontoverwaltung » Rollen » [Rolle auswählen]* legst du fest, welche Bereiche und Einstellungen Personen mit dieser Rolle sehen können, wenn sie ein Menü öffnen. Wenn die Rolle zum Beispiel keine Berechtigungen für ein Lager oder bestimmte Auftragsstatus hat, sind das Lager oder Aufträge in diesem Status für diese Person nicht sichtbar.
```

> Unter *Einrichtung » Kontoverwaltung » Rollen » [Rolle auswählen]* legst du fest, welche Bereiche und Einstellungen die Person mit der jeweiligen Rolle sehen kann, wenn die Person ein Menü öffnet. Bei fehlender Berechtigung ist zum Beispiel ein Lager oder sind Aufträge in einem Status für diese Person nicht sichtbar.

Reviewer 3: Ich würde mich im letzten Satz für ein Beispiel entscheiden, dann fuckt das die Grammatik nicht so sehr ab :)

> .Tab *Berechtigung* eines Benutzer:innen-Kontos vom Typ Backend

Reviewer: Ich dachte den Typ Backend gibt es nicht mehr?

Author: Gibt es auch nicht, die Stelle ist mir durch die Lappen gegangen

[cols="1,3"]
|====
|Einstellung |Erläuterung

| *Vertriebslager*; +
*Reparaturlager*; +
*Vorratslager*; +
*Umschlagslager*; +
*Verteilerlager*; +
*Sonderlager* (Typ *Sonstiges*) +
|Benutzer:innen Berechtigungen für Lager erteilen. Nur erstellte xref:warenwirtschaft:lager-einrichten.adoc#300[Lagertypen] stehen zur Verfügung. +

> *Alle* = Die Person mit diesem Benutzer:innen-Konto hat Zugriff auf alle Lager. +

Reviewer: ```suggestion
*Alle* = Personen mit dieser Rolle haben Zugriff auf alle Lager. +
```

*Keine* = Die Person mit diesem Benutzer:innen-Konto hat Zugriff auf keines der Lager.

| *Sichtbare Auftragsstatus*
|Auftragsstatus festlegen, die *sichtbar* sein sollen. Die Person mit diesem Benutzer:innen-Konto sieht nur die Aufträge, die in diesen Status sind.

| *Setzbare Auftragsstatus*

> |Auftragsstatus festlegen, die die Person mit diesem Benutzer:innen-Konto sehen darf.

Reviewer: ```suggestion
|Auftragsstatus festlegen, die Personen mit dieser Rolle sehen dürfen.
```

| *Ticketstatus*
|Ticketstatus wählen, um festzulegen, welche Tickets mit welchem Status die Person mit diesem Benutzer:innen-Konto ansehen und bearbeiten darf. +
*Unsichtbar* = Die Person mit diesem Benutzer:innen-Konto sieht kein Ticket, das in diesem Status ist. +
*Sichtbar* = Die Person mit diesem Benutzer:innen-Konto sieht jedes Ticket, das in diesem Status ist. +
*Nur eigene* = Die Person mit diesem Benutzer:innen-Konto sieht nur Tickets, die in diesem Status sind und von denen sie Eigner ist. +
In der Ticketübersicht werden alle Tickets aufgeführt, also auch für das Benutzer:innen-Konto unsichtbare Tickets. Die Person mit diesem Benutzer:innen-Konto kann unsichtbare Tickets jedoch nicht öffnen.
|====

[#120]
=== Menü- und REST-API-Rechte verwalten

Im erweiterten Modus legst du fest, welche Berechtigungen ein Benutzer:innen-Konto für die REST-API erhält. Benutzer:innen-Konten des Zugangs *Admin* verfügen über alle Berechtigungen.

[.instruction]
Menü- und REST-API-Rechte verwalten:

. Öffne das Menü *Einrichtung » Kontoverwaltung » Rollen*.
. Öffne eine Rolle oder erstelle eine neue.

> . Wechsel auf *Erweitert* (material:toggle_on[]).

Reviewer 2: ```suggestion
. Wechsele auf *Erweitert* (material:toggle_on[role=skyBlue]).
```

→ Die Tabelle des erweiterten Modus wird angezeigt.

> . Vergib die Berechtigungen in dem jeweiligen Bereich durch Anklicken der Checkboxen. Aktiviere komplette Bereiche oder klappe Bereiche auf, um einzelne Berechtigungen zu aktivieren, die dem Benutzer:innen-Konto zur Verfügung stehen sollen.

Reviewer 3: ```suggestion
. Vergib die Berechtigungen in dem jeweiligen Bereich durch Anklicken der Checkboxen. Aktiviere komplette Bereiche oder klappe Bereiche auf, um einzelne Berechtigungen zu aktivieren, die dem Benutzer:innen-Konto zugewiesen werden sollen.
```

. *Speichere* (icon:save[]) die Einstellungen.

[#125]
=== App-Berechtigungen aktivieren

> Unter dem Punkt *Mobile* im einfachen Modus der Rollenerstellung bzw. -bearbeitung legst du fest, welche Bereiche und Kennzahlen für die Rolle mit de in der plentymarkets App sichtbar sein sollen.

Reviewer 2: ```suggestion
Unter dem Punkt *Mobile* im einfachen Modus der Rollenerstellung bzw. -bearbeitung legst du fest, welche Bereiche und Kennzahlen für die Rolle in der plentymarkets App sichtbar sein sollen.
```

> Unter dem Punkt *Mobile* im einfachen Modus der Rollenerstellung bzw. -bearbeitung legst du fest, welche Bereiche und Kennzahlen für die Rolle mit de in der plentymarkets App sichtbar sein sollen.

Reviewer: ```suggestion
Unter dem Punkt *Mobile* im einfachen Modus der Rollen legst du fest, welche Bereiche und Kennzahlen für die Rolle in der plentymarkets App sichtbar sein sollen.
```

[.instruction]
Neue Rolle erstellen:

. Öffne das Menü *Einrichtung » Kontoverwaltung » Rollen*.

> . Klicke auf *Neu*. +

Reviewer 2: ```suggestion
. Klicke auf material:add[] *Neu*. +
```

> → Das Fenster *Neue Rolle* wird geöffnet.

Reviewer 2: der Satz kann mMn komplette weg.

. Gib einen *Namen* für die Rolle ein.
. *Speichere* (icon:save[]) die Einstellungen. +
→ Die Rolle wird in einem neuen Tab zur Bearbeitung geöffnet.

Erst beim Bearbeiten der Rolle legst du die Berechtigungen fest, die Benutzer:innen-Konten mit dieser Rolle haben. Bearbeite die Rolle deshalb nach dem Erstellen.

[#80]
== Rollen bearbeiten

> Nachdem du eine Rolle erstellt hast, wird sie zur Bearbeitung geöffnet und der Rollenübersicht hinzugefügt, so dass du die Rolle auch später wieder öffnen und bearbeiten kannst. Rollen werden in der Übersicht nach ID gelistet. Jede Rolle gliedert sich in vier Tabs. Im Tab *Einstellungen* lässt sich der Name der Rolle ändern. Im Tab *Rechte* ordnest du der Rolle im Einfachen Modus Menü-Sichtbarkeiten sowie Plugin-Sichtbarkeiten zu, im Erweiterten Modus die REST-API Berechtigungen. In den *Rollendefinierten Ansichten* lassen sich die verfügbaren My-View-Ansichten für eine Rolle anpassen. Es ist automatisch immer die Standardansicht ausgewählt. Unter *Zugrifssrechte* lassen sich die Ressouren aktivieren, auf die eine Rolle Zugriff haben soll. Details zu den Berechtigungen findest du hier auf der Seite im Kapitel xref:business-entscheidungen:benutzerkonten-zugaenge.adoc#105[Rechte verwalten].

Reviewer 2: ```suggestion
Nachdem du eine Rolle erstellt hast, wird sie zur Bearbeitung geöffnet und der Rollenübersicht hinzugefügt, sodass du die Rolle auch später wieder öffnen und bearbeiten kannst. Rollen werden in der Übersicht nach ID gelistet. Jede Rolle gliedert sich in vier Bereiche. Im Bereich *Einstellungen* kannst du den Namen der Rolle ändern. Im Bereich *Rechte* ordnest du der Rolle im Einfachen Modus Menü-Sichtbarkeiten sowie Plugin-Sichtbarkeiten zu, im Erweiterten Modus die REST-API Berechtigungen. Im Bereich *Rollendefinierte Ansichten* kannst du die verfügbaren MyView-Ansichten für eine Rolle anpassen. Es ist automatisch immer die Standardansicht ausgewählt. Im Bereich *Zugrifssrechte* lassen sich die Ressourcen aktivieren, auf die eine Rolle Zugriff haben soll. Details zu den Berechtigungen findest du hier auf der Seite im Kapitel xref:business-entscheidungen:benutzerkonten-zugaenge.adoc#105[Rechte verwalten].
```

> Nachdem du eine Rolle erstellt hast, wird sie zur Bearbeitung geöffnet und der Rollenübersicht hinzugefügt, so dass du die Rolle auch später wieder öffnen und bearbeiten kannst. Rollen werden in der Übersicht nach ID gelistet. Jede Rolle gliedert sich in vier Tabs. Im Tab *Einstellungen* lässt sich der Name der Rolle ändern. Im Tab *Rechte* ordnest du der Rolle im Einfachen Modus Menü-Sichtbarkeiten sowie Plugin-Sichtbarkeiten zu, im Erweiterten Modus die REST-API Berechtigungen. In den *Rollendefinierten Ansichten* lassen sich die verfügbaren My-View-Ansichten für eine Rolle anpassen. Es ist automatisch immer die Standardansicht ausgewählt. Unter *Zugrifssrechte* lassen sich die Ressouren aktivieren, auf die eine Rolle Zugriff haben soll. Details zu den Berechtigungen findest du hier auf der Seite im Kapitel xref:business-entscheidungen:benutzerkonten-zugaenge.adoc#105[Rechte verwalten].

Reviewer: Das ist keine Beschreibung, wie man Rollen bearbeitet. Hier fehlt eine Schrittliste. Die allgemeinen Infos, was man alles anpassen kann, sind ja schon weiter oben beschrieben.

> Rollen können von Benutzer:innen des Zugangs *Admin* erstellt werden oder von Nutzer:innen, die eine Rolle mit den notwendigen Rechten hat.

Reviewer: ```suggestion
Rollen können von Benutzer:innen des Zugangs *Admin* erstellt werden oder von Nutzer:innen, die eine Rolle mit den notwendigen Rechten haben.
```

Reviewer: Diese Info könnte außerdem deutlich weiter oben kommen, denn hier hat man ja schon längst Rollen erstellt.

[.instruction]
Rollen mit einem Benutzer:innen-Konto verknüpfen:

. Öffne das Menü *Einrichtung » Kontoverwaltung » Konten*.
. Öffne ein Benutzer:innen-Konto.

> . Klicke auf das Dropdown-Menü *Zugewiesene Rollen*

Reviewer: ```suggestion
. Klicke auf das Dropdown-Menü *Zugewiesene Rollen*.
```

[.instruction]
Rolle kopieren:

. Öffne das Menü *Einrichtung » Kontoverwaltung » Rollen*.
. Wähle eine Rolle.
. Klicke in der Zeile der Rolle auf material:more_vert[]. +
→ Ein Optionsfenster wird angezeigt.
. Wähle *Rolle duplizieren*. +

> → Die Rolle wird kopiert und mit dem Namen *Kopie von bisheriger Rolle* erstellt .

Reviewer 3: Lautet der neue Name buchstäblich "Kopie von bisheriger Rolle"?
Oder ist es "Kopie von [NAME DER KOPIERTEN ROLLE]?

Falls ersichtlich ist was ich meine?

[#60]
== Benutzer:innen-Konto suchen

> Filter helfen dir bei der Suche nach Benutzer:innen-Konten. Ohne Filter werden in der Übersicht alle Benutzer:innen-Konten angezeigt. Folgende Elemente und Funktionen sind im Menü *Konten* verfügbar.

Reviewer: ```suggestion
Filter helfen dir bei der Suche nach Benutzer:innen-Konten. Ohne Filter werden in der Übersicht alle Benutzer:innen-Konten angezeigt.
```

[.instruction]
Benutzer:innen-Konten suchen:

. Öffne das Menü *Einrichtung » Kontoverwaltung » Konten*.
. Gib in der Navigation *Filter* die gewünschten Filter ein.

> . Klicke auf Suchen (icon:search[]).

Reviewer: ```suggestion
. Klicke auf Suchen (icon:search[]). +
→ Die Ansicht wird gefiltert.
```
